apps/game/client/cl_contacts.ts

```diff
@@ -1,8 +1,72 @@
 import { ContactEvents } from '@typings/contact';
 import { RegisterNuiProxy } from './cl_utils';
 
+const QBCore = global.exports['qb-core'].GetCoreObject();
```

Comment:
As this is supposed to be as standalone as possible, we cannot use any third party library. We have these exports for external use only.